src/query/storages/fuse/src/io/segments.rs

```diff
@@ -143,11 +142,11 @@ impl SegmentsIO {
         });
 
         let threads_nums = self.ctx.get_settings().get_max_threads()? as usize;
-        let permit_nums = self.ctx.get_settings().get_max_storage_io_requests()? as usize;
+
```

Comment:
`SegmentsIO::write_segments` is no longer used, would you please remove this method?

Reply:
yes